README.md

```diff
@@ -1,6 +1,14 @@
 # AI:MMO
 A **M**assively **M**ulti-player **O**nline game, where players create **A**rtificially **I**ntelligent programs to play on their behalf.
 
+## A [Code for Life](https://www.codeforlife.education/) project
+* Ocado Technology's [Code for Life initiative](https://www.codeforlife.education/) has been developed to inspire the next generation of computer scientists and to help teachers deliver the computing curriculum.
+* This repository hosts the source code of the **AI:MMO game**.
```

Comment:
Bit short compared to the other repos, perhaps would be good to expand a bit on how AI:MMO fits in. Eg: 

> This repository hosts the source code of the **AI:MMO game**. AI:MMO is aimed as a follow-on from Rapid Router to teach secondary-school age children.